ui/component/fileRenderInline/index.js

```diff
@@ -1,34 +1,27 @@
 import { connect } from 'react-redux';
```

Comment:
"TextViewer" is now FileRenderInline

This is used for all content types that are renderable (i.e. not downloadable) but not "playable" (go into floating player)